homework_18/homework_18.py

```diff
@@ -0,0 +1,92 @@
+
+# 1. Створити клас Bot з атрибутом name та методами say_name та send_message.
+# send_message має приймати параметри self і message і має друкувати message.
+# Метод say_name має друкувати значення атрибуту name.
+
+class Bot:
+
+    def __init__(self):
+        self.name = "Bot"
```

Comment:
Bot should be able to take some name on initialization:
`some_bot = Bot('Marvin')`